inputs/templates/test/MainVcfQc/MainVcfQc_gq_filter.json.tmpl

```diff
@@ -0,0 +1,31 @@
+{
+  "MainVcfQc.primary_contigs_fai": {{ reference_resources.primary_contigs_fai | tojson }},
+
+  "MainVcfQc.site_level_comparison_datasets": [
+    {{ reference_resources.ccdg_abel_site_level_benchmarking_dataset | tojson }},
+    {{ reference_resources.gnomad_v2_collins_site_level_benchmarking_dataset | tojson }},
+    {{ reference_resources.hgsv_byrska_bishop_site_level_benchmarking_dataset | tojson }},
+    {{ reference_resources.thousand_genomes_site_level_benchmarking_dataset | tojson }}
+  ],
+  "MainVcfQc.sample_level_comparison_datasets": [
+    ["HGSV_ByrskaBishop", "gs://gatk-sv-resources-public/hg38/v0/sv-resources/resources/v1/HGSV_ByrskaBishop_GATKSV_perSample.tar.gz"]
+  ],
+  "MainVcfQc.sample_renaming_json": "gs://gatk-sv-resources-public/hg38/v0/sv-resources/resources/v1/rename_sample_ids_HGSV_ByrskaBishop.json",
+
+  "MainVcfQc.sv_pipeline_docker": {{ dockers.sv_pipeline_docker | tojson }},
+  "MainVcfQc.sv_base_mini_docker":{{ dockers.sv_base_mini_docker | tojson }},
+  "MainVcfQc.samtools_cloud_docker":{{ dockers.samtools_cloud_docker | tojson }},
+  "MainVcfQc.sv_pipeline_qc_docker": {{ dockers.sv_pipeline_qc_docker | tojson }},
+
+  "MainVcfQc.prefix": "HGDP_and_HGSV",
+  "MainVcfQc.ped_file": "gs://broad-dsde-methods-tbrookin/results/gq-filtering/inputs/HGDP_1KGP.ped",
+
+  "MainVcfQc.vcfs": [ "gs://broad-methods-cromwell-exec-bucket-instance-8/CrossValidateGqRecalibrator/749b2884-08c3-4b73-9c1e-da32c99c5d61/call-DirectGqRecalibrator/RecalibrateGq/50849ff4-3f30-4b02-829e-eb9b34ceea61/call-RecalibrateGqTask/cacheCopy/hgdp_and_hgsv.cleaned_fixed_annotated_gq_recalibrated.vcf.gz" ],
```

Comment:
I think it's cleaner to have one JSON template in general, but if this one has trios then maybe we should keep it as an alternative template to test the full breadth of functionality of MainVcfQc. Is this data able to be shared publicly? If so, can you make this VCF & PED file public? And can you move the VCF to a bucket without a TTL?

Also, can you rename the file to something like `MainVcfQc.WithTrios.json.tmpl` to be descriptive of what it contains & follow the naming convention of `WDLName.Descriptor.json.tmpl` (like GATKSVPipelineBatch.FromSampleEvidence.json.tmpl)?

Reply:
I renamed it to `MainVcfQc_per_sample_benchmark.json.tmpl`, because adding the per-sample benchmarking is the main difference. However, it's also using a different VCF and PED file than the other template too. Since I've had quite variable problems with memory and disk, I think it's possible the attribute overrides relate to those inputs.

Reply:
Sure, per_sample_benchmark is fine with me as an identifier. Can you still rename it to `MainVcfQc.per_sample_benchmark.json.tmpl` to match the `.` format used in other JSON templates in our repo?